docs/tutorials/services/try-viam-color-detection.md

```diff
@@ -49,20 +63,18 @@ Select the **Config** tab, and click on **Services**.
 Scroll to the **Create Service** section.
 To create a [vision service](/services/vision/):
 
-1. Select `vision` as the **Type**.
-2. Enter `my_color_detector` as the **Name**.
-3. Select **Color Detector** as the **Model**.
-4. Click **Create Service**.
+1. Select `Vision` as the **Type**.
+1. Enter `my_color_detector` as the **Name**.
+1. Select **Color Detector** as the **Model**.
+1. Click **Create Service**.
+1. In the resulting vision service panel, click the color picker box to set the color to be detected.
+   For this tutorial, set the color to `rgb(122, 79, 92)` or use hex code `#7a4f5c`.
 
-In your vision service's panel, fill in the **Attributes** field.
+1. Then, set **Hue Tolerance** to `0.06` and **Segment Size px** to `100`.
 
-``` json {class="line-numbers linkable-line-numbers"}
-{
-      "segment_size_px": 100,
-      "detect_color": "#7a4f5c",
-      "hue_tolerance_pct": 0.06
-}
-```
+Your configuration should look like the following:
+
+{{<imgproc src="/try-viam/try-viam/vision-service-config.png" resize="x500" alt="Screenshot of the vision service configuration showing the color set to a reddish color, the hue tolerance set to 0.06, and the segment size set to 100.">}}
```

Comment:
Please retake this screenshot and resize the window to a smaller width before you take it

Reply:
Thanks for this, you're right -- it's tough to read on mobile! Switched to a smaller width screenshot!